src/Core/GeneralAspectLoaderExtension.php

```diff
@@ -53,15 +52,19 @@ public function load(Aspect $aspect, ReflectionClass $reflectionAspect): array
         $loadedItems = [];
         foreach ($reflectionAspect->getMethods() as $aspectMethod) {
             $methodId    = $reflectionAspect->getName() . '->'. $aspectMethod->getName();
-            $annotations = $this->reader->getMethodAnnotations($aspectMethod);
+            $annotations = $aspectMethod->getAttributes(
```

Comment:
I guess, that `reader` property now can be removed and entire doctrine annotation reader too from the list of dependencies.

Reply:
`AbstractAspectLoaderExtension` can be patched too to remove reader

Reply:
Have you also implemented support for reading attributes in `goaop/parser-reflection`, as this part of code should work without touching real reflection, thus entire attribute support is required to be implemented via AST analysis only.

Reply:
You're right, some things can be removed now, I only aimed for a functioning base for the framework to support PHP 8

Reply:
I did some changes to the `goaop/parser-reflection`, but I am not really sure what I did there, all tests were passing though